### PR TITLE
Vue 1463 - Move some borrower profile prefetch

### PR DESCRIPTION
**Pull request by dyersituations**
https://kiva.atlassian.net/browse/VUE-1463

- I couldn't find usages of `state`, `description`, `isExpiringSoon`, or `reservedAmount`
- `my.userAccount` is used to populate the URLs/UTMs in `ShareButton`, so those were moved to `mounted`

### CODE REVIEW

**Review by eddieferrer — APPROVED** [review]
👍 good cleanup! 🧹

File: src/pages/BorrowerProfile/BorrowerProfile.vue [hunk 1]
```diff
@@ -212,7 +213,6 @@ const pageQuery = gql`
 				}
 				geocode {
 					city
-					state
```
Comment (mcstover) [inline src/pages/BorrowerProfile/BorrowerProfile.vue:215]: Is this not needed for comparison in @eddieferrer 's new background image work?

**Review by mcstover — APPROVED** [review]
Other than the comment about the state, which may be handled elsewhere? Looks good!

File: src/pages/BorrowerProfile/BorrowerProfile.vue (hunk 1 above)
Comment (dyersituations) [reply src/pages/BorrowerProfile/BorrowerProfile.vue:215]: `HeroBackground` has it's own query.

**Pull request merged**